runtime/gc_modron_startup/mminit.cpp

```diff
@@ -1089,6 +1089,15 @@ gcInitializeXmxXmdxVerification(J9JavaVM *javaVM, IDATA* memoryParameters, bool
 		}
 	}
 
+	/* Only if Xmx is specified, verify the value doesn't go over usable memory. */
```

Comment:
I apologize. I took one comment back. Would you please return back defined(J9ZTPF) wrapping for this section?

Reply:
@jjohnst-us Would you please fix last issues from two last comments and this will be good to go?